src/apis/storage_partitioning.js

```diff
@@ -95,6 +95,28 @@ export const partitioningSetEncrypt = ({ encrypt, partitioning }) => {
             });
 };
 
+/**
+ * @param {string} partitioning     DBus path to a partitioning
+ */
+export const partitioningSetHomeReuse = async ({ partitioning }) => {
+    const request = await getPartitioningRequest({ partitioning });
+
+    // In general default scheme defined in conf.Storage.default-scheme would be used
+    // Only btrfs (AUTOPART_TYPE_BTRFS) autopartitioning scheme is supported now
+    request["partitioning-scheme"] = cockpit.variant("i", 1);
```

Comment:
Shouldn't the partitioning scheme be set automatically by the backend, similarly how the scheme is currently set by the autopartitioning?

Reply:
Then we need to know the scheme for the request - as can be seen below, the request differs for various schemes. I'll create an update reading it from the conf and passing the scheme to this function.